package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asyncapi/server-api",
-  "version": "0.1.0",
+  "version": "0.0.1",
```

Comment:
As this version is already released it seems: https://github.com/asyncapi/server-api/releases/tag/v0.0.1

Would this be bumped automatically when releasing next version?

Reply:
Yes, it will be bumped, but semantic release check latest version and make bumps on that, so we have placeholde `0.0.1` release on Github to start next release - `0.1.0` :)